README.md

```diff
@@ -7,12 +7,65 @@ MyHebrewDates is a web app for sharing Hebrew dates like birthdays, anniversarie
 
 License: MIT
 
+## Contributors
+<div align="center">
+<a href="https://github.com/abe-101/myHebrewDates/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=abe-101/myHebrewDates" />
+</a>
+</div>
+
 ## Settings
 
 Moved to [settings](http://cookiecutter-django.readthedocs.io/en/latest/settings.html).
 
 ## Basic Commands
 
+## Installation
+
+1. Clone the repository:
+  ```shell
+  git clone https://github.com/abe-101/myHebrewDates.git
+  cd myHebrewDates
+  ```
+
+2. Create and activate a virtual environment:
+  ```shell
+  python3 -m venv venv
+  source venv/bin/activate
+  ```
+
+3. Install the local dependencies:
+  ```shell
+  pip install -r requirements/local.txt
+  ```
+
+4. Pre-Commit Install:
+  ```shell
+  pre-commit install
+  ```
+
+5. Create Database:
+  ```shell
+  createdb --username=<USERNAME> my_hebrew_dates
+  ```
+
+6. Create Env Files and add these:
```

Comment:
please change to 'Create `.env` Files and add these'

Reply:
yeah okay